download_action.go

```diff
@@ -102,6 +103,7 @@ func (d *DownloadAction) Run(context *DebosContext) error {
 		return fmt.Errorf("Incorrect filename is provided for '%s'", d.Url)
 	}
 	filename = path.Join(context.scratchdir, filename)
+	exportSymbol := filename
```

Comment:
nitpick i find that name confusing :) maybe make it originPath or somesuch?

Reply:
done